### DISCUSSION TITLE
6-item short form in jspsych

### DISCUSSION THREAD

**Discussion by Stajduhar-A** [jspsych/jsPsych#950] [Q&A] [answered]
Hi Josh,

I'm curious if there is any way to do something like this in jspsych: 

![Screen Shot 2020-08-04 at 8 50 41 PM](https://user-images.githubusercontent.com/52038520/89359578-2ad00080-d694-11ea-9b7c-8c8e04051fa4.png)

The participant should be able to click on the number they choose. Is there any way to do this? 

Thank you,
Andreja

**Comment by jodeleeuw** [comment, marked as answer]
Hi Andreja,

The `survey-likert` plugin will get you pretty close, though it won't look like this grid.

```js
var response_scale = ['Not at all', 'Somewhat', 'Moderately', 'Very Much'];
var likert = {
    type: 'survey-likert',
    questions: [
        {
          prompt: "I feel calm",
          labels: response_scale,
        },
        {
          prompt: "I am tense",
          labels: response_scale,
        },
        {
          prompt: "I feel upset",
          labels: response_scale,
        },
        {
          prompt: "I am relaxed",
          labels: response_scale,
        },
        {
          prompt: "I feel content",
          labels: response_scale,
        },
        {
          prompt: "I am worried",
          labels: response_scale,
        }
    ],
    scale_width: 500
};

```

It'll look like this:

![image](https://user-images.githubusercontent.com/595524/89426518-4117a400-d708-11ea-9fd0-f240470083a6.png)

You could also customize the plugin or create a new plugin to have a grid-based response.

**Reply by Stajduhar-A** [reply]
Hi Josh,

I did this with the survey-html-form, a very versatile little plugin :). 
<img width="1260" alt="Screen Shot 2020-08-05 at 7 23 06 PM" src="https://user-images.githubusercontent.com/52038520/89473535-19006300-d751-11ea-9c27-3c3308366b6f.png">

I'm just wondering how I would go about putting some extra padding between the bottom border of the table and the 'Continue' button. I tried adding a .divider tag to the CSS file and then putting a `<div>` tag with the divider class right before the button, but that didn't work. Thanks for all your help, Josh!

Best,
Andreja

**Reply by jodeleeuw** [reply]
Very nice!

What's the container element for the table that you've set up? You could probably add a `margin-bottom: 20px;` to that to add some space.

**Reply by Stajduhar-A** [reply]
That worked! Thanks Josh!
